tests/unit/Claim/ClaimsTest.php

```diff
@@ -237,38 +237,14 @@ public function testAddClaim() {
 		// Insert claim at the beginning:
 		$claim3 = $this->makeClaim( new PropertyNoValueSnak( new PropertyId( 'P17' ) ) );
 		$claims->addClaim( $claim3, 0 );
-		$this->assertEquals( 0, $claims->indexOf( $claim3 ), 'Inserting claim at the beginning failed' );
 
```

Comment:
I'm aware this this test method is not asserting a lot any more now. Since the code is deprecated, I suggest we do not waste time on fixing its bad tests though.